build/mp4-muxer.d.ts

```diff
@@ -201,4 +203,4 @@ declare global {
 }
 
 export { Muxer, MuxerOptions, ArrayBufferTarget, StreamTarget, FileSystemWritableFileStreamTarget };
```

Comment:
I'd also add the TransformationMatrix type as an export here!

Reply:
:+1: